### ISSUE TITLE
AGW: s1ap: test failed: test_multi_enb_multi_ue_diff_plmn

### ISSUE THREAD

**Issue by pshelar** [magma/magma#5645]
### Your Environment

- **Version:** master
- **Affected Component:** AGW
- **Affected Subcomponent:** MME
- **Deployment Environment:** e.g. Vagrant (AGW)

### Describe the Issue

S1AP test stuck:
vagrant@magma-test:~/magma/lte/gateway/python/integ_tests$ make integ_test TESTS=s1aptests/test_multi_enb_multi_ue_diff_plmn.py
. /home/vagrant/build/python/bin/activate
echo "Running test: s1aptests/test_multi_enb_multi_ue_diff_plmn.py"
Running test: s1aptests/test_multi_enb_multi_ue_diff_plmn.py
timeout -k 930s 900s sudo -E PATH=/usr/lib/ccache:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin PYTHONPATH=:/home/vagrant/s1ap-tester/bin /home/vagrant/build/python/bin/nosetests --with-xunit --xunit-file=/var/tmp/test_results/test_multi_enb_multi_ue_diff_plmn.xml -x -s s1aptests/test_multi_enb_multi_ue_diff_plmn.py || exit 1
Start time 04:38:07
tag: AUTH_TYPE tagVal: MILENAGE
************************* Enb tester config

**********************TPT open server******** 0

 Server Socket FD =[13]
************* Testing *Time elapsed(in usec): for timer expiry               :8896 usec
AGW is stateless
APN Correction configured
Health service is disabled
***************** Sending Multiple Enb Config Request

**** data->numOfEnbs = 5
Using subscriber IMSI IMSI001010000000001
Using subscriber IMSI IMSI001010000000002
Using subscriber IMSI IMSI001010000000003
Using subscriber IMSI IMSI001010000000004
Using subscriber IMSI IMSI001010000000005
************************* UE device config for ue_id  1
************************* UE device config for ue_id  2
************************* UE device config for ue_id  3
************************* UE device config for ue_id  4
************************* UE device config for ue_id  5
************************* Configuring IP block
************************* Waiting for IP changes to propagate
************************* S1 setup
---8<---
Its 100% reproducible on my setup.

**Comment by VinashakAnkitAman** [comment]
I see here from the description that the issue has been reproduced with the sanity test suite execution (make integ_test) only.

I ran the sanity on latest master magma (commit ID: 020dc923a94f9ce823cb75c432bcf1bc7714d6f5) multiple times and I could see sanity is passing every single time without any problem. For reference, I have attached S1APTester terminal log file for 2 continuous sanity executions. The sanity is not getting stuck for the mentioned test case for me.
[S1APTester-terminal-log-#5645.log](https://github.com/magma/magma/files/6183726/S1APTester-terminal-log-.5645.log)

@pshelar I would like to know more on your setup as I am not able to reproduce the issue.

I forgot to mention, so updating this point. This test case needs modification in the config file for multi PLMN changes. So I hope you have run the script to modify the configuration (make integ_test TESTS=s1aptests/test_modify_mme_config_for_sanity.py) before running this test case, if you are running this test case individually.
